pkg/controller/plan/migration.go

```diff
@@ -1596,6 +1645,128 @@ func (r *Migration) updateCopyProgressForOpenstack(vm *plan.VMStatus, step *plan
 	return
 }
 
+// Sets the OpenStackVolumePopulator CRs with VM ID and migration ID into the labels.
+func (r *Migration) setOpenStackPopulatorLabels(vm *plan.VMStatus, pvcs []core.PersistentVolumeClaim) (err error) {
+	openstackVm := &openstack.Workload{}
+	err = r.Source.Inventory.Find(openstackVm, vm.Ref)
+	if err != nil {
+		return
+	}
+	var images []*openstack.Image
+	for _, vol := range openstackVm.Volumes {
+		lookupName := fmt.Sprintf("%s-%s", r.Migration.Name, vol.ID)
+		image, err := r.getOpenstackImageByName(lookupName)
+		if err != nil {
+			continue
+		}
+		images = append(images, image)
+	}
+	if len(images) != len(pvcs) {
+		// To be sure we have every disk based on what already migrated and what's not.
+		// e.g when initialization the plan and we have populator CR without PVC or when the source changed.
```

Comment:
what does it mean "or when the source changed"?

Reply:
ok so how about making it more clear:
```suggestion
		// e.g when initializing the plan and the PVC has not been created yet (but the populator CR is) or when the disks that are attached to the source VM change.
```